### DISCUSSION TITLE
How to interpolate between images with BigGAN?

### DISCUSSION THREAD

**Discussion by Stmated** [deepjavalibrary/djl#1504] [Q&A]
I am quite new to all this, and is feeling quite lost.
How would I go about interpolating between two categories of images like in: https://colab.research.google.com/github/tensorflow/hub/blob/master/examples/colab/biggan_generation_with_tf_hub.ipynb#scrollTo=dSAyfDfnVugs

I can only figure out how to change the truncation, but not how anything else is handled.

I would assume it has something to do with a translator and changing some values, but even as a programmer of 15 years I don't understand what the Python code in that link is doing, or what to do inside the DJL library.

**Comment by frankfliu** [comment]
@Stmated 

Please take a look this blogpost: https://medium.com/@ql2282/generating-images-using-biggan-using-java-6e60ae7bac09

And the example in our repo: https://github.com/deepjavalibrary/djl/blob/master/examples/docs/biggan.md

You can find the Translator source code: https://github.com/deepjavalibrary/djl/blob/master/api/src/main/java/ai/djl/modality/cv/translator/BigGANTranslator.java
